src/daos/sql-query-builder.php

```diff
@@ -63,14 +67,18 @@ public function joins( $join_statement ) {
 
   /**
    * For setting a limit on the query other than the default
-   * @param $new_limit : integer
+   * To bind limit as int if passed as string use PDO binding parameter
+   * Example: select->bind_value( ':limit_to', array(intval( $options['limit'] ), \PDO::PARAM_INT ) )
+   *
+   * @param $new_limit
    */
   public function limit( $new_limit ) {
-    $this->limit = intval( $new_limit );
+    $this->limit = $new_limit;
```

Comment:
What happened to the intval ? that variable needs to be sanitized and instead of making everyone who uses the builder have to do it on their own we should enforce it here.